data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2142 @@
+		angle 90
+	turret -9 18 "Speckle Turret"
+	turret 9 18 "Speckle Turret"
+	turret 0 90 "Phased Array Laser (Green)"
+
+	leak "leak" 30 60
+	leak "flame" 30 90
+	explode "tiny explosion" 12
+	explode "small explosion" 15
+	explode "medium explosion" 15
+	"final explode" "final explosion medium"
+
+	description "The Prismaios-class Frigate is a recent, streamlined design, praised by many Twilight Guard captains as a highly successful addition to Avgi strike fleets. New construction techniques allow for unprecedented gains in structural integrity for such a lightweight frame, allowing the frigate to serve on the front lines against the Aberrant as a fast, versatile, multipurpose warship."
+
+
+ship "Interferos"
+	plural "Interferos"
+	sprite "ship/avgi interferos"
+	thumbnail "thumbnail/avgi interferos"
+	swizzle 0
+	attributes
+		category "Light Warship"
+		licenses
+			"Avgi Civilian"
+		"cost" 240000
+		"shields" 900
+		"hull" 3600
+		"automaton" 1
+		"crew equivalent" 3
+		"mass" 90
+		"drag" 1.5
+		"heat dissipation" 1.5
+		"fuel capacity" 600
+		"cargo space" 12
+		"outfit space" 282
+		"weapon capacity" 84
+		"engine capacity" 60
+		"anchor point" 1
+		weapon
+			"blast radius" 210
+			"shield damage" 900
+			"hull damage" 450
+			"hit force" 900
+	outfits
+		"Phased Array Laser (Green)" 2
+		"Sidereal Launcher" 4
+		"Sidereal Missile" 168
+
+		"Luminous Fusion Core"
+		"Ultracapacitor Cell" 3
+		"Diffuse Deflector"
+		"Optical Lasing Generator"
+		"Electrostatic Ramscoop"
+		
+		"Z-360 Fusion Torch"
+		"XY-180 Radial Thrusters"
+		"Hyperdrive"
+
+	gun -33 -57 "Sidereal Launcher"
+		angle -60
+	gun 33 -57 "Sidereal Launcher"
+		angle 60
+	gun -33 -42 "Sidereal Launcher"
+		angle -60
+	gun 33 -42 "Sidereal Launcher"
+		angle 60
+	turret -21.5 -81 "Phased Array Laser (Green)"
+	turret 21.5 -81 "Phased Array Laser (Green)"
+	turret 0 -48
+
+	engine 0 99
+	"reverse engine" -6 99
+		angle -30
+	"reverse engine" 6 99
+		angle 30
+	"steering engine" -9 -108
+		angle 90
+		right
+	"steering engine" 9 -108
+		angle -90
+		left
+	"steering engine" -6 99
+		angle 90
+		left
+	"steering engine" 6 99
+		angle -90
+		right
+
+	leak "flame" 60 90
+	explode "tiny explosion" 15
+	explode "small explosion" 9
+	explode "medium explosion" 6
+	"final explode" "final explosion medium"
+
+	description "The Interferos-class Frigate is an autonomous support ship, designed to be easy to build and disposable enough for mass production. To free up front line warships from having to escort merchant convoys in the Tangles, the Consonance commissioned this design with the intent of licensing its sale to independent merchant captains. It has since become popular due to its impressively large armament, though its attrition rates leave something to be desired."
+
+
+# Medium Warships
+
+ship "Diaspersi"
+	sprite "ship/avgi diaspersi"
+	thumbnail "thumbnail/avgi diaspersi"
+	swizzle 0
+	attributes
+		category "Medium Warship"
+		licenses
+			"Avgi Military"
+		"cost" 2400000
```

Comment:
```suggestion
		"cost" 3600000
```
This carries more ships than comparable human carriers (Aerie two fighters at 3.5M, Roost four fighters at 3M), but does have less weapon space (96 vs 150 for the Aerie or 140 for the nest/roost/skein), hence a similar but slightly higher cost.